### ISSUE TITLE
Position toast in the center of the screen

### ISSUE THREAD

**Issue by sunviwo** [jossmac/react-toast-notifications#138]
Can toast be positioned in the center of the screen?

**Comment by asoutheastasiandessert** [comment]
We have the same issue.

**Comment by jossmac** [comment]
The [available placements](https://github.com/jossmac/react-toast-notifications#toastprovider-props) are documented, so you could place the toast `'bottom-center'` or `'top-center'`.

If the desired placement is right in the middle of the viewport you could [replace the `ToastContainer`](https://github.com/jossmac/react-toast-notifications/blob/master/README.md#replaceable-components) component.

**Comment by asoutheastasiandessert** [comment]
Hi Joss, I believe it is due to the minimum width set to 300px, causing phones with a narrower mobile view to have the toast notifications truncated. 
https://user-images.githubusercontent.com/73132254/100313487-b4169480-2fef-11eb-8c1b-37523e2282a6.png

**Comment by asoutheastasiandessert** [comment]
I think it would be good if you can set 80% width of the screen (80vw) and have a maxwidth of 300px.  https://user-images.githubusercontent.com/73132254/100313697-1b344900-2ff0-11eb-9e18-8c69210cf672.png)

**Comment by jossmac** [comment]
@asoutheastasiandessert you can target the toast with the class `react-toast-notifications__toast`

**Comment by robertn702** [comment]
FYI the following appropriately fixed the width on smaller screens (e.g. iPhone 5)

```javascript
const CustomToastContainer = styled(DefaultToastContainer)`
  max-width: 100%;
`;

const CustomToast = styled(DefaultToast)`
  max-width: 100%;
`;
```

```javascript
<ToastProvider placement="bottom-center" components={{
  Toast: CustomToast,
  ToastContainer: CustomToastContainer,
}}>
```

**Comment by jossmac** [comment]
Resolved in `2.4.1`
